pkg/minikube/tunnel/tunnel.go

```diff
@@ -136,17 +141,53 @@ func setupRoute(t *tunnel) {
 
 	if !exists && len(conflict) == 0 {
 		t.status.RouteError = t.router.EnsureRouteIsAdded(t.status.TunnelID.Route)
-		if t.status.RouteError == nil {
-			//the route was added successfully, we need to make sure the registry has it too
-			//this might fail in race conditions, when another process created this tunnel
-			if err := t.registry.Register(&t.status.TunnelID); err != nil {
-				glog.Errorf("failed to register tunnel: %s", err)
-				t.status.RouteError = err
+		if t.status.RouteError != nil {
+			return
+		}
+		//the route was added successfully, we need to make sure the registry has it too
+		//this might fail in race conditions, when another process created this tunnel
+		if err := t.registry.Register(&t.status.TunnelID); err != nil {
+			glog.Errorf("failed to register tunnel: %s", err)
+			t.status.RouteError = err
+			return
+		} else if h.DriverName == "hyperkit" {
+			//the virtio-net interface acts up with ip tunnels :(
+			command := exec.Command("ifconfig", "bridge100")
+			fmt.Printf("About to run command: %s\n", command.Args)
+			response, _ := command.CombinedOutput()
+			iface := string(response)
+			pattern := regexp.MustCompile(`.*member: (en\d+) flags=3<LEARNING,DISCOVER>.*`)
+			submatch := pattern.FindStringSubmatch(iface)
+			if len(submatch) != 2 {
+				//error
+				t.status.RouteError = fmt.Errorf("error, couldn't find member in bridge100 interface: \n%s", iface)
```

Comment:
Remove "error," - as including "error," in the string this far down makes it quite likely that the final error message seen by the user will have "error" mentioned multiple times. 

Also, no need for a newline in this error message.